app/src/main/java/org/thoughtcrime/securesms/BaseActionBarActivity.java

```diff
@@ -97,7 +97,7 @@ protected void onCreate(Bundle savedInstanceState) {
   protected void onResume() {
     super.onResume();
     initializeScreenshotSecurity(true);
-    DynamicLanguageActivityHelper.recreateIfNotInCorrectLanguage(this, TextSecurePreferences.getLanguage(this));
+    DynamicLanguageActivityHelper.recreateIfNotInCorrectLanguage(this, MessagingModuleConfiguration.getShared().getPrefs().getLanguage());
```

Comment:
Is it worth putting a... what's the word, um, `convenience getter` (? I can't think of it - but what I'm after is _syntactic sugar_!) on this so we can just call `MessagingModuleConfiguration.getLanguage()` or something?

Same for anything else that calls `MessagingModuleConfiguration.getShared().getPrefs()` before it's final "give me the thing".

It looks like you have this further down where you just call `context.prefs.<something>` - is that the same thing?

Reply:
Yeah, though ideally this is injected, I'll see if I can do that.

haha, yeah `context.prefs` should be the same thing, but again, if `prefs` is injected, that's more correct.

I'll look into it.